opencti-platform/opencti-front/src/utils/hooks/useGranted.ts

```diff
@@ -58,4 +58,9 @@ const useGranted = (capabilities: string[], matchAll = false): boolean => {
   return numberOfAvailableCapabilities > 0;
 };
 
+export const useIsAdmin = () => {
+  // only admins have access to the platform settings, so it's a good shortcut for now
+  return useGranted([SETTINGS]);
+};
```

Comment:
woops ! I used it for something that I removed during the development.

Right now it's not used anywhere.
We can keep it anyway... it's still a good idea to have a dedicated hook I think 🤔

I can do a quick search-replace to refactor this in the platform.

Reply:
I pushed a commit for that; only in the cases where the code was explicitly looking for "isAdmin".

There are other places around using the grant on SETTINGS but semantically it might be better to leave it for code clarity.